lib/app/group.js

```diff
@@ -20,9 +20,23 @@ var membership = [
 ];
 
 module.exports = function(mediator) {
-  var groupStore = new ArrayStore('group', groups);
-  groupStore.listen('cloud:', mediator);
+  var groupStore = new SimpleStore('group');
+  groupStore.list().then(function(res) {
```

Comment:
`res` is a bit generic. Can you rename these to be more descriptive (e.g. `groupList` )